pkg/kubectl/kubectl.go

```diff
@@ -172,3 +173,26 @@ func CreateSecretFromLiteral(c Client, namespace, name, key, value string, overr
 
 	return true, nil
 }
+
+type secretManifest struct {
+	Data map[string]string
+}
+
+// GetSecretValue returns the value of a secret
+func GetSecretValue(c Client, namespace, name, key string) (string, error) {
```

Comment:
One thing we could do to test these functions is to have special test `Client` that return a sample `kubectl get secret ...` output string so we can test we decoding logic.